### PR TITLE
OADP-3788: Adding that IBM Cloud Object Storage S3 is supported

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [OADP-3788](https://issues.redhat.com/browse/OADP-3788)

### VERSION

* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15
* OCP 4.16 → branch/enterprise-4.16

### Link to docs preview:

* [About installing OADP](https://73981--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/oadp-installing-dpa-1-3.adoc
Comment (anarnold97) [inline modules/oadp-installing-dpa-1-3.adoc:96]:
```suggestion
<10> Specify whether to force path style URLs for S3 objects (Boolean). Required only for {ibm-cloud-name} Object Storage S3.
```

File: modules/oadp-installing-dpa-1-3.adoc
Comment (anarnold97) [inline modules/oadp-installing-dpa-1-3.adoc:96]:
```suggestion
<10> Specify whether to force path style URLs for S3 objects (Boolean). Not Required for AWS S3. Required only for S3 compatible storage .
```

File: modules/oadp-installing-dpa-1-3.adoc
Comment (anarnold97) [inline modules/oadp-installing-dpa-1-3.adoc:97]:
```suggestion
<11> Specify the URL of the object store that you are using to store backups. Not required for AWS S3. Required only for S3 compatible storage.
```

File: modules/oadp-installing-dpa-1-3.adoc
Comment (anarnold97) [inline modules/oadp-installing-dpa-1-3.adoc:96]:
```suggestion
<10> Specify whether to force path style URLs for S3 objects (Boolean). Not Required for AWS S3. Required only for S3 compatible storage.
```

**Review by PrasadJoshi12 — APPROVED** [review]

**Review by skopacz1 — COMMENTED** [review]
Left a few comments but otherwise LGTM!

**Comment by skopacz1** [comment]
So unfortunately while the PR LGTM, since this feature hasn't been mentioned before in the already published docs you want to CP to, this is essentially like backporting a feature to published docs, which requires change management to be done. 

I would hold onto this PR until that's done, but this likely won't be done until next week, by which point I won't be on the review squad. So I'm dropping the merge review labels, but please put this PR back into the queue once Change Management has been done. Let me know if you have any questions, thanks!

**Comment by anarnold97** [comment]
@kalexand-rh - this has been approved by Dev and QE, please can we have it merged?

**Comment by anarnold97** [comment]
Included in 1.3.1 release notes [Here](https://github.com/openshift/openshift-docs/pull/74026/files)

**Comment by anarnold97** [comment]
Changed version to 4.13+

Will need to make a second PR for 1.2 DPA

**Comment by nmuller66** [comment]
Hey @anarnold97, the issue that Seb called out is that backporting new functionality not previously released/documented  to older versions requires change management (as documented in the OpenShift Docs Manual). 

From what I understand, this functionality will be released in the GA  of  1.3.1 on 4-16-24. 
There's a note in the OS Docs Manual that "Change management is not required for changes that are introduced along with the release of a new product version." However, even in this scenario, you need to work with the engineering teams to make that you have a Docs-ACK at the PgM level before backporting. 

I see **2 options** - either pursue the documented Change Management process, or wait to backport alongside the feature GA, making sure you have the appropriate approvals in place for the backported content.

Please take a look at the docs manual and consider what might be the best path forward for this PR. I know the team is happy to answer questions and help find a path if you have concerns or more context to share.

**Comment by anarnold97** [comment]
Ok so please can I ask if I leave it until Monday to push to merge review, you will not have an issue, is this correct? 

We are not a large team, in fact we are a very small team who have 4 releases next week: OADP 1.3.1, MTR 1.2.5, MTC 1.8.3, MTV 2.6.0, and we still have a lot to do and get through merge review. Obviously, I am a bit concerned it could create a bottleneck if we have issues. Especially when we have larger PRs to get through QE review, for example [PR#72372](https://github.com/openshift/openshift-docs/pull/72372).

**Comment by anarnold97** [comment]
But if those are the rules, those are the rules

**Comment by nmuller66** [comment]
@skopacz1 @bergerhoffer 

Can you answer @anarnold97 here?

I shared with Andy the idea of holding this PR to release alongside the functionality GA on 4-16 instead of going through CM. Andy has some concerns about the load of release-related work his team has coming up in the next week. 
> We are not a large team, in fact we are a very small team who have 4 releases next week: OADP 1.3.1, MTR 1.2.5, MTC 1.8.3, MTV 2.6.0, and we still have a lot to do and get through merge review. Obviously, I am a bit concerned it could create a bottleneck if we have issues. Especially when we have larger PRs to get through QE review, for example [PR#72372](https://github.com/openshift/openshift-docs/pull/72372).

I think it would be a good idea for @anarnold97 to share this information in the #forum-ocp-docs-review and @peer-review-squad Slack channels to help the review squads to know that there is urgency around these PRs. 
Are any other ways to get this PR, or others, ready to merge - even if they don't actually merge until GA? Or, is it best to just notify the squads of the urgency at the beginning of the week through the aforementioned Slack channels?

Thank you!

**Comment by skopacz1** [comment]
I agree with @nmuller66's two options mentioned in this comment: https://github.com/openshift/openshift-docs/pull/73981#issuecomment-2052064405

> share this information in the [#forum-ocp-docs-review](https://redhat-internal.slack.com/archives/C025XCRRH28) and @peer-review-squad Slack channels to help the review squads to know that there is urgency around these PRs.

+1 to this, I think it would be a good idea to ping the new squad next week in [#forum-ocp-docs-review](https://redhat-internal.slack.com/archives/C025XCRRH28) explaining the situation and urgency.

**Review by sheriff-rh — APPROVED** [review]
Changes LGTM, merging to 4.13+.

**Comment by sheriff-rh** [comment]
Merged 4.14-16, but you'll need a manual cherrypick for 4.13 @anarnold97. Tag me when you have it ready and I'll review/merge it. :rocket:

**Pull request merged**
